### ISSUE TITLE
Towny Support - Not working

### ISSUE THREAD

**Issue by lukemango** [Wolfst0rm/ArmorStandEditor-Issues#15]
### Expected behavior

Players not able to use in other towns

### Observed/Actual behavior

Players can use this plugin anywhere

### Steps/models to reproduce

Create a town, not be in the town on another account and try to edit an armor stand

### Plugin list

AdvancedEnchantments, AnCasino, antiRedstoneClock, ArmorStandEditor, AttributeDemo*, BetonQuest, BetterRTP, BuycraftX, ChatFilter, ChatReaction, ChestShop, ChestShopNotifier, Chunky, Citizens, CMILib, CommandPanels, CommandWatcher*, CoreProtect, CountdownPlugin, CraftingStore, CrateReloaded, CrazyEnvoy, CYTAuctions, CYTAuctionWorth, CYTAutoSmelt, CYTBalTop, CYTBiomeChange, CYTChatPing, CYTCore, CYTDemonTower, CYTGearStats, CYTLasso, CYTLottery, CYTNoGrindstones, CYTWheel, DeluxeCommands, DeluxeMenus, DiscordSRV, EcoPower, Essentials, EssentialsChat, EssentialsSpawn, EternalTags, ExoticGarden, ExperienceBottle*, ExtraTools, FarmLimiter, FastAsyncWorldEdit (WorldEdit), FeatherBoard, FluffyMachines, ForceResourcepacks (WorldResourcepacks), GuiRedeemMCMMO*, Harbor, HeadDatabase, HeadDisplays, HolographicDisplays, iConomy, Images, Infiniteannouncements, InfinityExpansion, IPWhitelist*, JetsAntiAFKPro, JetsRepairTokens, Jobs, LeaderHeadsRevamped, LiteBans, LiteXpansion, LuckPerms, LWC, mcMMO, MineSecure*, Multiverse-Core, Multiverse-Inventories, Multiverse-Portals, MythicCrucible, MythicDrops, MythicMobs, ntdLuckyBlock, OpenInv, PinataParty, Pl3xMap, Pl3xMap-Towny, PlaceholderAPI, Plan, PlayerVaults, PlayerWarps, PlugMan, ProCosmetics, ProtocolLib, PyroFishingPro, Quests, Rankup, SFCalc, SimpleRename, Slimefun, SlimyTreeTaps, spark, SuperVanish, TAB, Towny, TownyChat, TradePlus, UltimateVotes, UltraBar, UpgradeableSpawners, Vault, VoidGen, VoteFly, VoteParty, Votifier, WorldGuard, WorldGuardExtraFlags, XLTournaments, XPBoost, XRayTracker

### Plugin Version

1.18-pre5-30

### Server Version

git-Pufferfish-4 (https://ci.pufferfish.host/job/Pufferfish-Purpur/) - Fork of Airplane

**Comment by Wolfieheart** [comment]
WIP -> Investigating

**Comment by Wolfst0rm** [comment]
Tested using the following

**Plugin Version (ASE):** 
```
[17:10:37 INFO]: [ArmorStandEditor] ======= ArmorStandEditor =======
[17:10:37 INFO]: [ArmorStandEditor] Plugin Version: 1.18-rc3-31
[17:10:37 INFO]: [ArmorStandEditor] SpigotMC: true
[17:10:37 INFO]: [ArmorStandEditor] PaperMC: true
[17:10:37 INFO]: [ArmorStandEditor] Minecraft Version: v1_17_R1 is supported. Loading continuing.
[17:10:37 INFO]: [ArmorStandEditor] ================================
```

**Towny Plugin Version:**
```
[17:10:39 INFO]: =============================================================
[17:10:39 INFO]: [Towny] Version: 0.97.3.0 - Plugin Enabled
[17:10:39 INFO]: =============================================================
```

**User with the following permissions:**

```
[17:30:43 INFO]: [LP] (2 additions and 0 deletions)
[17:30:43 INFO]: +  asedit.equipment (true)
[17:30:43 INFO]: +  asedit.basic (true)
```

Unable to successfully reproduce when account 1 was in the town, other user was invited and in the Wilderness (PVP). Could be permissions related etc.

**Comment by Wolfieheart** [comment]
Confimed - Users cant place armorstands outside towns in Wilderness. Checking.

**Comment by Wolfieheart** [comment]
Could be something related to EditTool being set and ignoring certain Towny Protections. Investigating Further

**Comment by Wolfieheart** [comment]
Ok figured out the issue. Fix will be apart of the 1.18 update :D 
Thanks for letting me know this does not work!
